lib/srv/desktop/rdp/rdpclient/src/rdpdr.rs

```diff
@@ -190,6 +190,10 @@ impl Client {
                 }
             };
 
+            if self.test_debug_logs {
```

Comment:
I feel like there's got to be a better way to enable debugging in tests other than putting everything behind a flag.

Paging @espadolini @xacrimon - you guys have any suggestions?

Reply:
This isn't actually a log that I want when running tests, its just a temporary log I want to help me build the tests. Once I'm done with this round of testing I'm planning to remove the flag. Definitely a hack.